apps/web/src/server/applications/lib/services/case.service.js

```diff
@@ -101,3 +102,40 @@ export const updateCase = async (caseId, payload) => {
 export const getCaseDocumentationCategories = (id) => {
 	return get(`applications/${id}/documents`);
 };
+
+/**
+ * Get the parent folder(s) for the current folder - used for breadcrumbs etc.
+ *
+ * @param {number} caseId
+ * @param {DocumentationCategory |undefined} documentationCategory
+ * @returns {DocumentationCategory[]}
+ */
+export const getCaseDocumentationFolderTree = (caseId, documentationCategory) => {
+	// TODO:Mock Version:
+	/** @type {DocumentationCategory[] } */
+	let folderTree = [];
+
+	if (caseId && documentationCategory) {
+		folderTree = [documentationCategory];
+	}
+	return folderTree;
+	// TODO: when API method created: return get(`applications/${caseId}/folder/${folderId}/parent-folders`);
+};
+
+/**
+ * Get the documents for the current folder
+ *
+ * @param {number} caseId
+ * @param {DocumentationCategory |undefined} documentationCategory
+ * @returns {DocumentationFile[]}
+ */
+export const getCaseDocumentationFilesInFolder = (caseId, documentationCategory) => {
```

Comment:
not sure what you're using documentationcategory for here

Reply:
it holds details of the current folder we are viewing, and will be needed in order to return the list of files in that folder from the API when it is created